lxd/storage_volumes_utils.go

```diff
@@ -200,6 +201,10 @@ func storagePoolVolumeUsedByContainersGet(s *state.State, volumeName string,
 		}
 	}
 
+	if len(ctsUsingVolume) == 0 {
+		return []string{}, os.ErrNotExist
```

Comment:
That seems wrong to me, it looks to me like the pool doesn't exist when in fact it's just that we don't have any volume, why isn't an empty list the right answer here?

Reply:
I'd prefer notexist errors but I'm fine the other way too.